ci/before_install.sh

```diff
@@ -44,7 +44,6 @@ fi
 
 source ~/.nvm/nvm.sh
 nvm install 10
```

Comment:
Future idea: perhaps only the "gcov" job should install ruby/python/nodejs/perl. That will make CI (1) more reliable and (2) perhaps save 30-60 seconds per job.